includes/api/class-wc-admin-rest-setting-options-controller.php

```diff
@@ -16,35 +16,10 @@
  * @extends WC_REST_Setting_Options_Controller
  */
 class WC_Admin_REST_Setting_Options_Controller extends WC_REST_Setting_Options_Controller {
-
 	/**
 	 * Endpoint namespace.
 	 *
 	 * @var string
 	 */
 	protected $namespace = 'wc/v4';
-
-	/**
-	 * Attempt to separate by comma and validate, otherwise send to parent function.
-	 *
-	 * @since 3.0.0
-	 * @param array $values Values.
-	 * @param array $setting Setting.
-	 * @return array|WP_Error
-	 */
-	public function validate_setting_multiselect_field( $values, $setting ) {
-		$final_values = array();
-		$split_values = explode( ',', $values );
-
-		foreach ( $split_values as $value ) {
-			if ( array_key_exists( $value, $setting['options'] ) ) {
-				$final_values[] = $value;
-			} else {
-				return parent::validate_setting_multiselect_field( $values, $setting );
-			}
-		}
-
-		return $final_values;
-	}
-
-}
+}
```

Comment:
Per WP's PHP coding standards & linter rules, PHP files must end with a newline character. I think this is causing Travis to fail as well.